modules/ui/OptimizerParamsWindow.py

```diff
@@ -123,7 +123,8 @@ def create_dynamic_ui(
             'no_prox': {'title': 'No Prox', 'tooltip': 'Whether to use proximity updates or not.', 'type': 'bool'},
             'optim_bits': {'title': 'Optim Bits', 'tooltip': 'Number of bits used for optimization.', 'type': 'int'},
             'percentile_clipping': {'title': 'Percentile Clipping', 'tooltip': 'Gradient clipping based on percentile values.', 'type': 'float'},
-            'r': {'title': 'r', 'tooltip': 'Use polynomial weighting in the average with power r', 'type': 'float'},
+            # Repeated key in `dict`
```

Comment:
@Nerogar should we delete this line entirely? It was never active since it was shadowed by the later "r" entry overwriting it.

Reply:
I think it'd be good to keep around in case we did move to a framework that could display this tooltip or renamed the existing variable somehow.

Reply:
I think we can remove it. If we add support for separate tooltips for each optimizer, we will need to go through all parameters anyway. There is no point in keeping this one specifically.